static/js/util/currency.js

```diff
@@ -1,17 +1,50 @@
 // @flow
 import cc from 'currency-codes';
 import R from 'ramda';
+import iso3166 from 'iso-3166-2';
+
+export const excludedCurrencyCodes = [
```

Comment:
This list may change over the course of time, depending on whether the frontend or backend APIs decide they want to add or remove supported currencies.  Our backend automatically updates our list of supported currencies based on what the backend API returns to us, which means `CurrencyExchangeRate.objects.values_list('currency_code' , flat=True)` will always return the list of supported and current exchange rates.

I don't think it's that likely that the list of supported/not supported codes will change too quickly, but fetching these dynamically instead of hardcoding them would protect against those cases.